### PR TITLE
Fix org.eclipse.search.textSearchEngine ext point definition

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

**Comment by mickaelistria** [comment]
@robstryker Please have a look, this works for me.

**Comment by mickaelistria** [comment]
I noticed https://ci.eclipse.org/platform/job/eclipse.platform.ui/job/PR-1122/5/console succeeded to build (and then timed out later). Let's merge.
I also see timeout failures for master, the org.eclipse.e4.tools.test seem to sometimes take ages to complete. We'll probably have to increase the timeout later or do something smarter about it. But for now, let's unblock things.

File: tests/org.eclipse.search.tests/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="org.eclipse.search.textSearchEngine">
+      <textSearchEngine
+            class="org.eclipse.search.tests.TestTextSearchEngine"
```
Comment (iloveeclipse) [inline tests/org.eclipse.search.tests/plugin.xml:7]: I see an **error** in the IDE now: `Referenced class 'org.eclipse.search.tests.TestTextSearchEngine' in attribute 'class' is not on the plug-in classpath`.

Can we fix that please?

Comment (mickaelistria) [reply tests/org.eclipse.search.tests/plugin.xml:7]: This is now fixed with https://github.com/eclipse-platform/eclipse.platform.ui/pull/1137

**Pull request merged**